Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
Dmitri Zagidulin <notifications@github.com> writes:
> Is there a way to get these annotated tags to generate different HTML
> tags? Like, a way to specify that `@@quote` should generate an HTML
> `<blockquote>`?

that's unfortunately not possible

-- 
Damien Cassou
http://damiencassou.seasidehosting.st

"Success is the ability to go from one failure to another without
losing enthusiasm." --Winston Churchill